test/testHTMLUniversal.js

```diff
@@ -702,7 +702,7 @@ ${context.content.document.body.innerHTML}`,
     assert.ok(res.body.match(/<img/));
   });
 
-  it('html.pipe makes HTTP requests and prefers branch param for surrogate computation', async () => {
+  it.skip('html.pipe makes HTTP requests and prefers branch param for surrogate computation', async () => {
     const myparams = { ...params };
     myparams.branch = 'mybranch';
```

Comment:
@trieloff if you would create a `mybranch` in your soup demo, this should work again.